.gitignore

```diff
@@ -243,3 +243,5 @@ RealmWeaver.Fody.dll
 wrappers/libs-dbg
 
 *.xam
+
+Tests/IntegrationTests.XamarinAndroid/Resources/Resource.designer.cs
```

Comment:
Yes in spirit but after discussion on Slack, would rather we ignore it and remove it everywhere.